src/util/gobConfig.ml

```diff
@@ -303,10 +305,16 @@ struct
     in
     drop memo_int; drop memo_bool; drop memo_string; drop memo_list
 
-  let get_int    = memo_int.get
-  let get_bool   = memo_bool.get
-  let get_string = memo_string.get
-  let get_list   = memo_list.get
+  let wrap_get f x =
+    (* self-observe options, which Spec construction depends on *)
+    (* if !building_spec then
+      Printf.printf "GET during spec building: %s\n" x; *)
```

Comment:
Commented out? How about removing it or converting it to a tracing?

Reply:
I think it might not be too bad to have it in the beginning such that we can then limit what can be changed in the server mode, this whitelisting is still completely missing, allowing for all sorts of dangerous changes to be made.